cinema/tests/test_movie_view_set.py

```diff
@@ -0,0 +1,201 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient
+
+from cinema.models import Movie, Genre, Actor
+from cinema.serializers import MovieSerializer, MovieListSerializer, MovieDetailSerializer
+
+MOVIE_URL = reverse("cinema:movie-list")
+
+def detail_url(movie_id):
+    return reverse("cinema:movie-detail", args=[movie_id])
+
+def sample_movie(**params) -> Movie:
+    defaults = {
+        "title": "The Great Brick",
+        "description": "The Great Brick fall on superman head",
+        "duration": 121
+    }
+    defaults.update(params)
+    return Movie.objects.create(**defaults)
```

Comment:
This is a code quality issue. You're using the `**` operator to unpack the dictionary and pass its items as keyword arguments to the `create` method. This is a good practice as it makes the code cleaner and easier to read.